edgexfoundry-holding/device-onvif-camera
# feat: Update config toml to add the default profile

Using the "./res/profiles" as profileDir to provide the default device profile for the user.

Closes #2

Signed-off-by: bruce <weichou1229@gmail.com>







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/IOTechSystems/device-onvif-camera/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] I have added unit tests for the new feature or bug fix (if not, why?) **not impact the unit test**
  <link to docs PR>

## Testing Instructions

1. Run core services
2. Run device-onvif-camera
3. Execute the following command to check whether the device profile exist.
   `http://localhost:59881/api/v2/deviceprofile/all`

## New Dependency Instructions (If applicable)
